app/assets/javascripts/discourse/app/components/d-button.js

```diff
@@ -96,7 +97,14 @@ export default Component.extend({
 
   @discourseComputed("ariaExpanded")
   computedAriaExpanded(ariaExpanded) {
-    return ariaExpanded ? "true" : "false";
+    if (isPresent(ariaExpanded)) {
```

Comment:
If you're using `===` to compare with `true` and `false`, `isPresent` isn't needed, is it?

Reply:
Yes it's not needed anymore